building/tracks/ci/gha-best-practices.md

```diff
@@ -0,0 +1,140 @@
+# GitHub Actions: Best Practices
+
+This _working_ document serves as a collection of best practices for making use of GitHub Actions.
+If you have any suggestions or additions, [please open a pull request on GitHub!](https://github.com/exercism/docs/edit/main/building/tracks/ci/gha-best-practices.md)
+
+- [Collection of Best Practices](#collection-of-best-practices)
+  - [Set timeouts for workflows](#set-timeouts-for-workflows)
+  - [Consider if (third-party) actions are really needed](#consider-if-third-party-actions-are-really-needed)
+  - [Limit scope of workflow token](#limit-scope-of-workflow-token)
+  - [Pin actions to SHAs](#pin-actions-to-shas)
+  - [Consider setting up a concurrency strategy](#consider-setting-up-a-concurrency-strategy)
+  - [Consider which triggers are really needed](#consider-which-triggers-are-really-needed)
+  - [Read the "Security hardening for GitHub Actions" guide](#read-the-security-hardening-for-github-actions-guide)
+- [Workflow Checklist](#workflow-checklist)
+
+## Collection of Best Practices
+
+### Set timeouts for workflows
+
+By default, GitHub Actions kills workflows after 6 hours if they have not finished by then.
+Many workflows don't need nearly as much time to finish, but sometimes unexpected errors occur or a job hangs until the workflow run is killed 6 hours after starting it.
+Therefore it's recommended to [specify a shorter timeout](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes).
+
+The ideal timeout depends on the individual workflow but 30 minutes is typically more than enough for the workflows used in Exercism repos.
+
+This has the following advantages:
+
+- PRs won't be pending CI for half the day, issues can be caught early or workflow runs can be restarted.
```

Comment:
6 hours is a quarter of a day.  (Just sayin' 😉)

Reply:
It's roughly half of the day if you exclude sleep and such ;)